jwst/pipeline/outlier_detection.cfg

```diff
@@ -14,4 +14,4 @@ scale = '0.5 0.4'
 backg = 0.0        
 save_intermediate_results = False
 resample_data = True
-good_bits = 6
+good_bits = 'SATURATED, JUMP_DET'
```

Comment:
We should be consistent with what's in the step spec, in other words set these to ~DO_NOT_USE.